src/WebDocs/wwwroot/app/data/SectorContainerDataTwo.html

```diff
@@ -0,0 +1,16 @@
+﻿<!DOCTYPE html>
+<script src="/drapo.js"></script>
+<div d-sector-parent-url="/DrapoPages/SectorContainerData.html" d-sector-parent="sector">
+    <span>2</span>
+    <div>
+        <div d-dataKey="users" d-dataUrlGet="/Data/GetModelSample"></div>
```

Comment:
Controller data does not exist

Reply:
I did comparisons, using "GetModelSample and 'CreateNodes', both had the same result in the sample, so I used CreateNodes, so we don't need to add GetModelSample.